packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -96,24 +96,24 @@ export default {
 		},
 		isDecrementEnabled() {
 			return (
-				this.currentValueNumber !== null &&
-				!this.disabled &&
-				isInRange({
-					max: null,
-					min: this.min,
-					value: this.currentValueNumber - this.step,
-				})
+				this.currentValueNumber === null ||
```

Comment:
Looking good @carsoli , one suggestion:

I think we need to guard against this.disabled always and then add the range and null check logic, currently a disabled component has active increment/decrement buttons.
`
isDecrementEnabled() {
    if (this.disabled) return false

    this.currentValueNumber === null || isInRange({
 						max: null,
 						min: this.min,
 						value: this.currentValueNumber - this.step,
 					}))

}
`